.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```

Comment:
I'm going to add `.vscode` to the `.gitignore` since `.vscode` is only related to the editor and not our project so, remove this file from the PR.